kpi/models/import_export_task.py

```diff
@@ -516,10 +517,10 @@ def _run_task(self, messages):
         # Take this opportunity to do some housekeeping
         self.log_and_mark_stuck_as_errored(self.user, source_url)
 
-        if hasattr(source.deployment, '_get_submissions'):
+        if isinstance(source.deployment, MockDeploymentBackend):
```

Comment:
[Now that `build_formpack()` uses `get_submissions()`](https://github.com/kobotoolbox/kpi/pull/1864/files#diff-f173fc2ca93b2e54a20782253da31ffeR105), could we omit this condition altogether?

(not blocking)

Reply:
Sure, but some part of the code already exists in `gallery-2`. 
Let's do it then both branches are merged.